README.md

```diff
@@ -22,14 +22,14 @@ Improving the Developer Experience at ESDC through building a culture of Continu
 
 ## Strategy [WIP]
 
-Use Dashboards to measure progress over time as well as to provide instant feedback to developers to allow spontaneous learning opportunites.
+Use Dashboards to measure progress over time as well as to provide instant feedback to developers to allow continous learning opportunites.
 
-Put in place enterprise wide systems (ex: Secret Management, Dependency Managment, etc...) that enables developers to increase the quality of their systems without expending a great deal of effort. These systems should be audited against ITSG-33 Compliance controls so that just by using these tools systems are compliant thus reducing the compliance burden of teams and increasing the compliance posture of the organization as a whole. 
+Establish enterprise wide systems (ex: Secret Management, Dependency Managment, etc...) in colaberation with key teams (SADE, IT-Sec) that enables developers to increase the quality of their products without expending a great deal of effort. Products should be audited against ITSG-33 Compliance controls so that by using these systems, products are compliant reducing the burden on teams and increasing the compliance posture of the organization as a whole. 
```

Comment:
Collaboration is spelled incorrectly. 

and it's the "Enterprise wide Systems" we want to be "assessed"